CASMCMS-8720: Update docs for CFS v3 and paging (#4283)
Cray-HPE/docs-csm: operations/README.md

The Configuration Framework Service \(CFS\) is available on systems for remote execution and configuration management of nodes and boot images.

- [Configuration Management](configuration_management/Configuration_Management.md)

>   - [CFS Cheat Sheet](configuration_management/CFS_Cheatsheet.md)

Reviewer: If you rename the file per the other comment, then this should be updated
```suggestion
  - [CFS Commands Cheat Sheet](configuration_management/CFS_Commands_Cheat_Sheet.md)
```

  - [CFS Flow Diagrams](configuration_management/CFS_Flow_Diagrams.md)
  - [CFS Global Options](configuration_management/CFS_Global_Options.md)

>   - [Differences in the new V3 API](configuration_management/V2_vs_V3.md)

Reviewer: If you rename this file per my other comment, then this link should be updated.
```suggestion
  - [Differences Between the V2 and V3 CFS APIs](configuration_management/Differences_Between_the_V2_and_V3_CFS_APIs.md)
```

  - [Paging CFS Records](configuration_management/Paging_CFS_Records.md)
  - [Automatic Configuration Management](configuration_management/Automatic_Configuration_Management.md)
  - [CFS Key Management](configuration_management/CFS_Key_Management.md)
- [Managing CFS Configurations](configuration_management/CFS_Configurations.md)
- [Managing CFS Components](configuration_management/CFS_Components.md)

> - [Managing CFS Sessions](configuration_management/CFS_Sessions.md)

Reviewer: ```suggestion
- [CFS Configurations](configuration_management/CFS_Configurations.md)
- [CFS Components](configuration_management/CFS_Components.md)
- [CFS Sessions](configuration_management/CFS_Sessions.md)
```
